tools/macos_buildenv.sh

```diff
@@ -35,7 +34,7 @@ case "$COMMAND" in
 
         if [[ "$BUILDENV_NAME" =~ .*macosminimum([0-9]*\.[0-9]*).* ]]
         then
-            MACOSX_DEPLOYMENT_TARGET="${BASH_REMATCH[1]}"
+            export MACOSX_DEPLOYMENT_TARGET="${BASH_REMATCH[1]}"
```

Comment:
I was going back and forth about this. For local macOS development there is no need to set this. A developer putting this script in their shell profile might not want this variable set for other projects. On the other hand, it ensures that the same environment is used locally as on CI.

Reply:
I think it's worth having the same setup. Otherwise it might be difficult to reproduce bugs locally when they were reported by people using the CI artifacts, right?

I woudn't put this script into my shell profile, because it's only needed for Mixxx development. To avoid accidently setting this variable when writing code for other projects, we could make it explicit if you're using the mixxx buildenv or not by preprending something to `$PS1`:

    $

becomes

    [mixxxenv] $

or something like that (similar to python venvs).

Reply:
> To avoid accidently setting this variable when writing code for other projects, we could make it explicit if you're using the mixxx buildenv or not

I think that would be annoying for most developers. I hope that the few for whom this environment variable would interfere would know that. Come to think of it, they should set this variable differently anyway when working on another project with a different requirement.